homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -43,9 +44,20 @@ class LaCrosseSensorEntityDescription(
     """Description for LaCrosse View sensor."""
 
 
-def get_value(sensor: Sensor, field: str) -> float:
+def get_value(sensor: Sensor, field: str) -> float | int | str | None:
     """Get the value of a sensor field."""
-    return float(sensor.data[field]["values"][-1]["s"])
+    field_data = sensor.data.get(field)
+    if field_data is None:
+        LOGGER.warning(
```

Comment:
`get_value` will be called in an entity state property. We don't want side effects like logging in entity state properties.

Please either remove the logging or move it to a state update callback.